components/core/src/clp/NetworkReader.cpp

```diff
@@ -10,6 +10,7 @@
 
 #include <curl/curl.h>
 
+#include "Array.hpp"
```

Comment:
This include isn't needed in this file, is it?

Reply:
true, removed.